yapapi/rest/activity.py

```diff
@@ -40,12 +40,8 @@ async def new_activity(self, agreement_id: str) -> "Activity":
                  and allows to query and control its state
         :rtype: Activity
         """
-        try:
-            activity_id = await self._api.create_activity(agreement_id)
-            return Activity(self._api, self._state, activity_id)
-        except yexc.ApiException:
-            _log.error("Failed to create activity for agreement %s", agreement_id)
-            raise
+        activity_id = await self._api.create_activity(agreement_id)
```

Comment:
No need to log anything here, the caller of this method handles errors and prints them with traceback to the debug log (as part of `ActivityCreateFailed` event).